benchmark/test/reference/spmv_distributed.profile.stdout

```diff
@@ -5,7 +5,7 @@
         "comm_pattern": "stencil",
         "spmv": {
             "csr-csr": {
-                "storage": 11476,
+                "storage": 11452,
```

Comment:
It is from DenseCache -> ScalarCache, we allocate the memory when using it.
It is 3 * sizeof(ValueType) because we use 3 processes for mpi